_test/test_admin/test_upgrade_file_format.m

```diff
@@ -52,7 +52,7 @@ function test_upgrade_single_sqw_filebacked_noupgrade_range_warning(obj)
             w2  = sqw(targ_f{1});            
 
             [~,e] = lastwarn;
-            assertEqual(e,'HORACE:invalid_data_range');
+            assertEqual(e,'TESTS:my_warning');
```

Comment:
because the part of the job I am doing is to make data range more reliable and remove unnecessary warnings when the range is incorrect. It has been done for upgrade_file_format, which I am using.